src/ui/public/vislib/components/color/colormaps.js

```diff
@@ -17,2739 +17,2777 @@
-  [0.477, [0.985, 0.443, 0.317]],
-  [0.479, [0.985, 0.443, 0.317]],
-  [0.481, [0.985, 0.438, 0.312]],
-  [0.483, [0.985, 0.438, 0.312]],
-  [0.485, [0.985, 0.433, 0.307]],
-  [0.487, [0.985, 0.433, 0.307]],
-  [0.489, [0.985, 0.428, 0.302]],
-  [0.491, [0.985, 0.428, 0.302]],
-  [0.493, [0.984, 0.423, 0.298]],
-  [0.495, [0.984, 0.423, 0.298]],
-  [0.497, [0.984, 0.418, 0.293]],
-  [0.499, [0.984, 0.418, 0.293]],
-  [0.501, [0.984, 0.413, 0.288]],
-  [0.503, [0.984, 0.413, 0.288]],
-  [0.505, [0.982, 0.407, 0.285]],
-  [0.507, [0.982, 0.407, 0.285]],
-  [0.509, [0.981, 0.401, 0.281]],
-  [0.511, [0.981, 0.401, 0.281]],
-  [0.513, [0.979, 0.395, 0.277]],
-  [0.515, [0.979, 0.395, 0.277]],
-  [0.517, [0.978, 0.390, 0.274]],
-  [0.519, [0.978, 0.390, 0.274]],
-  [0.521, [0.976, 0.384, 0.270]],
-  [0.523, [0.976, 0.384, 0.270]],
-  [0.524, [0.975, 0.378, 0.266]],
-  [0.526, [0.975, 0.378, 0.266]],
-  [0.528, [0.973, 0.372, 0.263]],
-  [0.530, [0.973, 0.372, 0.263]],
-  [0.532, [0.972, 0.367, 0.259]],
-  [0.534, [0.972, 0.367, 0.259]],
-  [0.536, [0.970, 0.361, 0.255]],
-  [0.538, [0.970, 0.361, 0.255]],
-  [0.540, [0.969, 0.355, 0.251]],
-  [0.542, [0.969, 0.355, 0.251]],
-  [0.544, [0.967, 0.349, 0.248]],
-  [0.546, [0.967, 0.349, 0.248]],
-  [0.548, [0.966, 0.343, 0.244]],
-  [0.550, [0.966, 0.343, 0.244]],
-  [0.552, [0.964, 0.338, 0.240]],
-  [0.554, [0.964, 0.338, 0.240]],
-  [0.556, [0.963, 0.332, 0.237]],
-  [0.558, [0.963, 0.332, 0.237]],
-  [0.560, [0.961, 0.326, 0.233]],
-  [0.562, [0.961, 0.326, 0.233]],
-  [0.564, [0.960, 0.320, 0.229]],
-  [0.566, [0.960, 0.320, 0.229]],
-  [0.568, [0.958, 0.314, 0.226]],
-  [0.569, [0.958, 0.314, 0.226]],
-  [0.571, [0.957, 0.309, 0.222]],
-  [0.573, [0.957, 0.309, 0.222]],
-  [0.575, [0.956, 0.303, 0.218]],
-  [0.577, [0.956, 0.303, 0.218]],
-  [0.579, [0.954, 0.297, 0.215]],
-  [0.581, [0.954, 0.297, 0.215]],
-  [0.583, [0.953, 0.291, 0.211]],
-  [0.585, [0.953, 0.291, 0.211]],
-  [0.587, [0.951, 0.286, 0.207]],
-  [0.589, [0.951, 0.286, 0.207]],
-  [0.591, [0.950, 0.280, 0.203]],
-  [0.593, [0.950, 0.280, 0.203]],
-  [0.595, [0.948, 0.274, 0.200]],
-  [0.597, [0.948, 0.274, 0.200]],
-  [0.599, [0.947, 0.268, 0.196]],
-  [0.601, [0.947, 0.268, 0.196]],
-  [0.603, [0.945, 0.262, 0.192]],
-  [0.605, [0.945, 0.262, 0.192]],
-  [0.607, [0.944, 0.257, 0.189]],
-  [0.609, [0.944, 0.257, 0.189]],
-  [0.611, [0.942, 0.251, 0.185]],
-  [0.613, [0.942, 0.251, 0.185]],
-  [0.614, [0.941, 0.245, 0.181]],
-  [0.616, [0.941, 0.245, 0.181]],
-  [0.618, [0.939, 0.239, 0.178]],
-  [0.620, [0.939, 0.239, 0.178]],
-  [0.622, [0.938, 0.234, 0.174]],
-  [0.624, [0.938, 0.234, 0.174]],
-  [0.626, [0.934, 0.229, 0.171]],
-  [0.628, [0.934, 0.229, 0.171]],
-  [0.630, [0.930, 0.224, 0.170]],
-  [0.632, [0.930, 0.224, 0.170]],
-  [0.634, [0.926, 0.220, 0.168]],
-  [0.636, [0.926, 0.220, 0.168]],
-  [0.638, [0.921, 0.216, 0.166]],
-  [0.640, [0.921, 0.216, 0.166]],
-  [0.642, [0.917, 0.211, 0.164]],
-  [0.644, [0.917, 0.21
```

Comment:
I can get rid of wrapping constants into function after the PR #23902 is merged.